var/spack/repos/builtin/packages/jali/package.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Jali(CMakePackage):
+    """Jali is a parallel unstructured mesh infrastructure library
+designed for use by multi-physics simulations. It supports 2D and 3D
```

Comment:
Should probably indent this. Also, is this entire description necessary? Most packages have one or two sentences.

Reply:
Done.